readme.md

```diff
@@ -3,16 +3,16 @@
 
 Build badtracksecrets:
 
-`python3 ./build_secrets.py -u <Email Username> -p <Email Password>`
+`EMAIL_USER=\<username> EMAIL_PASSWORD=\<password> ./build_secrets.py
```

Comment:
what's with the backslash?

Reply:
markdown escape character. WIthout it the <username> and <password> wouldn't display